### PR TITLE
Refs [Task] [47846] Tạo footer

### PR DESCRIPTION
**Pull request by vuhang1712**
https://edu-redmine.sun-asterisk.vn/issues/47846

### CODE REVIEW

**Comment by vuhang1712** [comment]
image

**Comment by vuhang1712** [comment]
ready

File: project2/src/components/Footer/Footer.jsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+function Footer() {
+  return (
+    <footer className="footer">
+      <Container>
+        <Row>
+          <Col md={3}>
+            <h2 className="footer__title">LIÊN HỆ VỚI LAZADA</h2>
+            <div className="footer__list">
+              <Link to="/">Hotline & Chat trực tuyến (24/7)</Link>
```
Comment (vietnq-1097) [inline project2/src/components/Footer/Footer.jsx]: nav ul li nhé

**Comment by vuhang1712** [comment]
ready

**Pull request merged**